WooCommerce/Classes/ViewRelated/Dashboard/Settings/Settings/SettingsViewController.swift

```diff
@@ -813,6 +824,11 @@ private extension SettingsViewController {
             comment: "Navigates to Install Jetpack screen."
         )
 
+        static let themes = NSLocalizedString(
+            "Themes",
+            comment: "Navigates to Themes screen."
+        )
```

Comment:
Let's use the new style of localizing strings with keys and values.

```suggestion
        static let themes = NSLocalizedString(
            "settingsViewController.themesRow",
            value: "Themes",
            comment: "Navigates to Themes screen."
        )
```

Reply:
Thanks!